Sources/pam-watchid/pam_watchid.swift

```diff
@@ -80,7 +80,7 @@ private func parseArguments(argc: Int, argv: vchar) -> [String: String] {
 
 private extension LAPolicy {
     static var deviceOwnerAuthenticationIgnoringUserID: LAPolicy {
-#if SEQUOIASDK
+#if compiler(>=6.0)
```

Comment:
Usually the swift used is the one installed as part of Xcode, but it's also possible to install a newer version of the toolchain manually from [swift.org](https://swift.org/). This would allow the 6.0 compiler to be used on older versions of macOS which don't support the required SDK. The current version of swift is being built to support versions of macOS as old as Big Sur. This causes errors like this:

<img width="977" alt="Screenshot 2024-12-20 at 11 57 54 pm" src="https://github.com/user-attachments/assets/b752973e-607c-47bf-a88f-5fbfab77593b" />

I know it's an absurdly niche environment, but it's technically a regression and not something I want to potentially have to deal with later.

As an alternative, `canImport()` could be used to check if a newer framework is available. For example [ImagePlayground](https://developer.apple.com/documentation/imageplayground) was added in macOS 15.1 according to the docs. I haven't had a good look so there might be something better suited but if not, I'm happy to set the cutoff at 15.1 as it should still be compatible.

Reply:
I was aware of that problem, yeah, but I couldn't find any frameworks that were introduced specifically in 15.0, and if I checked with ImagePlayground, then this package would fail to build on macOS 15.0..<15.1, which I thought was a bigger sacrifice to make than this niche scenario

Reply:
Maybe I could combine the checks?

Reply:
aha! [this site](https://marcoeidinger.github.io/appleframeworks/) was my savior, and I can check `#if canImport(CoreHID)`